src/view/dataElements/xdmObject/components/noSelectedNodeView.jsx

```diff
@@ -47,50 +44,39 @@ const NoSelectedNodeView = props => {
       <div>
         <p>
           Build an object that complies with your configured schema by selecting
-          attributes on the left and providing their values here.
-        </p>
-        <p className="u-flex">
-          <PopulationAmountIndicator
-            populationAmount={EMPTY}
-            className="u-gapRight NoSelectedNodeView-icon"
-          />
-          An empty circle like this indicates no attributes have been filled in.
-        </p>
-        <p className="u-flex">
-          <PopulationAmountIndicator
-            populationAmount={PARTIAL}
-            className="u-gapRight NoSelectedNodeView-icon"
-          />
-          A partially filled in circle like this indicates some of the
-          attributes have been filled in.
-        </p>
-        <p className="u-flex">
-          <PopulationAmountIndicator
-            populationAmount={FULL}
-            className="u-gapRight NoSelectedNodeView-icon"
-          />
-          A full circle like this indicates all of the attributes have been
-          filled in.
-        </p>
-        <p className="u-flex">
-          <span className="u-flex">
-            <AsteriskIcon
-              className="IconTip-icon u-gapRight NoSelectedNodeView-icon"
-              size="XS"
-            />
-          </span>
-          Fields that may be auto-populated when this data element is passed to
-          the XDM option of the &quot;Send Event&quot; action have this icon.
-          Hovering over the icon shows a popup explaining when the field will be
-          auto-populated.
+          attributes on the left and providing their values.
         </p>
+        <Flex direction="column" gap="size-100">
+          <IndicatorDescription
+            indicator={<PopulationAmountIndicator populationAmount={EMPTY} />}
+          >
+            An empty circle indicates no attributes have been populated.
+          </IndicatorDescription>
+          <IndicatorDescription
+            indicator={<PopulationAmountIndicator populationAmount={PARTIAL} />}
+          >
+            A partially filled in circle indicates some of the attributes have
+            been populated.
+          </IndicatorDescription>
+          <IndicatorDescription
+            indicator={<PopulationAmountIndicator populationAmount={FULL} />}
+          >
+            A full circle indicates all of the attributes have been populated.
+          </IndicatorDescription>
+          <IndicatorDescription indicator={<AsteriskIcon size="XS" />}>
+            Fields that may be auto-populated when this data element is passed
+            to the XDM option of the &quot;Send Event&quot; action have this
+            icon. Hovering over the icon shows a popup explaining when the field
+            will be auto-populated.
+          </IndicatorDescription>
+        </Flex>
```

Comment:
We could probably avoid a lot of our utility classes by using React-Spectrum's Flex and View components. I'm still feeling that out.